docs/components/concepts/process-instance-migration.md

```diff
@@ -0,0 +1,206 @@
+
+<!--
+I'm using process, process model, and process definition interchangeably, because I don't know what's best.
+Using it interchangeably might help users pick up an understanding of what we mean (some may know it as a model, others as a process).
+But, it's also inconsistent.
+Should we use all terms or should we select a specific one.
+The API refers to it as the "process instance's process definition" and the "target process definition".
+-->
+
+To achieve this, we can migrate our process instance from its current process to the newly deployed one. You must provide a migration plan with mapping instructions to the target process definition to clarify your intentions.
+
+In our example, we'll provide a mapping instruction from source element ID `A` to target element ID `A`. This means we expect any active element instances of element with ID `A` to be migrated to the element with ID `A` in the target process.
+Therefore, our process instance will continue to be active at service task `A`.
+
+<!-- TODO: add screenshot of the migrated process instance -->
+
+After migrating our process instance, we can now complete the service task `A` to reach user task `B`.
+
+Process instance migration allows you to change the inactive parts of the process instance. In our example, we placed a user task `B` between the active service task `A` and the inactive end event. We did not change the active service task `A`, just the steps that follow.
+
+## Changing the active elements
+
+Sometimes your requirements change so much that the currently active element no longer exists in the new process version.
+
+Consider the following example: the process contains a completed exclusive gateway, taking the sequence flow to the service task `A` which is currently active.
+It did not take the sequence flow to service task `B`, so it is inactive.
+
+<!-- TODO: add screenshot of a process instance with a completed XOR gateway, an active service task A and an inactive service task B -->
+
+Due to changed requirements, our process model no longer contains the exclusive gateway, nor the service tasks `A` and `B`.
+Instead, it only contains a new service task `C`.
+
+<!-- TODO: add screenshot of a prcoess with only service task C -->
+
+We can migrate the active service task `A` to this new service task `C` by providing a mapping instruction from source element ID `A` to target element ID `C`.
+
+:::note
+You cannot map an element to an element of a different type.
+An active service task can only be mapped to an active service task.
+It cannot be mapped to a user task as this changes the element type.
+
+Also note that the [jobs, expressions, and input mappings](#jobs-expressions-and-input-mappings) are not recreated.
+So, while service task `C` is active in the target process, the associated job still has the job type from when it was associated to service task `A`.
+:::
+
+<!--
+Should we even have this section 'Changing the active elements' at all?
```

Comment:
I am all about trimming up a doc if the information isn't necessarily beneficial. I would remove this, or just add a note summarizing (which you kind of already do below).